### ISSUE TITLE
Web.Control subclass defined without a namespace does not work

### ISSUE THREAD

**Issue by granicz** [dotnet-websharper/core#1206]
I have the following web control:

```fsharp
module Controls =
    open WebSharper.UI.Client

    module Server =
        [<Rpc>]
        let GetData () = 
            async { return [1; 2; 3] }

    type MyControl() =
        inherit Web.Control()

        [<JavaScript>]
        override this.Body =
            let model = ListModel.FromSeq []
            async {
                let! data = Server.GetData()
                model.AppendMany(data)
            }
            |> Async.Start
            ul [] [
                model.Doc(fun i -> li [] [text ("Value = " + string i)])
            ] :> _
```

which I use in a sitelet as:

```fsharp
    Sitelet.Infer (fun ctx -> function
        | Home ->
            MainTemplate()
                .Body(
                    div [] [
                        Doc.WebControl <| Controls.MyControl()
                    ]
                )
                .Doc()
            |> Content.Page
...
```

However, the resulting page has no trace of `MyControl` and nothing is shown in its place.

**Comment by Jand42** [comment]
@granicz I have tested this exact code in a Client-Server template created with WebSharper.Templates 5.0.0.102-preview1.
Could not reproduce any issue.

![image](https://user-images.githubusercontent.com/3116993/134812898-b8e1afba-6b65-45e6-82c4-4f901b5a929b.png)

**Comment by Jand42** [comment]
Turns out issue is only when `Web.Control` subclass is not within a namespace. (F#'s `module X` on top level of a file will translate to static class `X` without namespace). Renaming ticket accordingly.

**Comment by Jand42** [comment]
Tested original failing project, fixed with `getName` fix in cfbc4b0
